### ISSUE TITLE
Crashes Pokemon Crystal JP version

### ISSUE THREAD

**Issue by marbuljon** [Drenn1/GameYob#140]
Hey, when playing the Japanese Pokemon Crystal, when you go into the Goldenrod pokecenter the game stops working and shows an error message. When I try to change the settings (the ones like "recognize this as a GBA") and then try going inside, instead of getting that message the game just flashes the inside of the pokecenter for a brief moment and then freezes. I'm playing on a 3DS if that matters any. For now I'm just avoiding that pokecenter entirely (and I haven't found any other bugs). Here's the in-game error:

このカートリッジはゲームボーイカラーせんようです。ゲームボーイカラーでしようしてください。

It seems this is the English version of the error, with some info about it:
http://glitchcity.info/wiki/index.php/This_Game_Pak_is_designed_only_for_use_on_the_Game_Boy_Color

Also there's another minor issue that doesn't need to be fixed, but for some reason Crystal is saving improperly. It works fine to save using the "save states" from the emulator, but saving in-game doesn't really work, and often when I start up the game from its normal menu and not by defrosting the save state, the game is messed up like I'm in a different spot and a different gender from what/where I was supposed to be.

**Comment by marbuljon** [comment]
I don't know how to edit what I wrote so I'll leave it here: The error is inescapable, as in, once you reach that error screen you just have to restart the game in order to get out of it.

**Comment by AlbertGiHub** [comment]
I have this problem too @@

**Comment by AlbertGiHub** [comment]
I've try same rom with lameboy and VisualBoyAdvance on PC, Both of them are no problem here.

**Comment by Jimmy-Z** [comment]
I'm playing the English version, not problem so far, except for link transfer.

**Comment by marbuljon** [comment]
Yes, because the English version was re-coded a bit and excludes some features that the Japanese version had.
